portalnet/src/discovery.rs

```diff
@@ -425,7 +430,7 @@ impl AsyncUdpSocket<UtpEnr> for Discv5UdpSocket {
 
 // todo: remove this once sigp/enr implements this for enr's
 // we need this because signatures can be different for the same data but still valid
-fn verify_by_signature(enr: &Enr, signature: &[u8]) -> bool {
+fn get_enr_rlp_content(enr: &Enr) -> BytesMut {
```

Comment:
@KolbyML I don't think there is any harm in exposing `rlp_content` from `Enr`

Reply:
When we get the update Enr version we can just use compare_content. If we want to expose rlp_content() that is fine too. It just looked like rlp_content() was intentionally privated so that is why I tried to get a compare* function added. Cause I didn't want to manually calculate rlp_content in multiple codebases.

Anyways up to you. I didn't know making rlp_content() was an option tbh.

Reply:
I do think compare_content is cleaner though
``enr1.compare_content(&enr2)``
vs
``enr1.rlp_content() == enr2.rlp_content()``

Reply:
Neither is a problem. Nothing wrong with exposing it, this does not change the enr and it's not private info in any way